rules.gradle

```diff
@@ -26,9 +26,9 @@ dependencies {
 // Check if the android plugin version supports unit testing.
 if (configurations.findByName("testCompile")) {
```

Comment:
I think this probably needs to be updated as well. Essentially the idea here is to avoid declaring test dependencies if the android plugin version being used doesn't handle them. But if "testCompile" will be removed in the future, this check will start failing.

We can change to "testImplementation" for now. I think we're probably fine to do some broader cleanup here - the fact that we don't just declare these dependencies directly is some legacy where we'd import Volley source code into another build system, and I don't believe there's anyone actively depending on that today. But I don't want to overcomplicate the starter bug :)

Reply:
Thanks! I just made the change in the line you highlighted.